### ISSUE TITLE
prefer IPv4 over IPv6 tweak broke postgresql

### ISSUE THREAD

**Issue by Susukerow45** [ChrisTitusTech/winutil#3090]
## Describe the bug
After applying the tweak, connection to local PostgreSQL on 127.0.0.1 databases stops working and an error appears related to the pg_hba.conf file.

## Steps to reproduce
1. Go to tweaks tab
2. Click on prefer IPv4 over IPv6 in Essential tweaks 
3. Click on Run tweaks
4. Restart PC 
5. See the error in programs that use postgresql local base on 127.0.0.1

## Expected behavior
Just working 

## Additional context
Move tweak "prefer IPv4 over IPv6" to "Advanced tweaks - CAUTION" category so ppl have no fafo

**Comment by ChrisTitusTech** [comment]
Agreed, I'm going to remove this. As more applications start leaning more on ipv6 it is causing more issues than it solves.

**Comment by MyDrift-user** [comment]
It was moved to advanced Tweaks some time ago. Therefore I am going to close this out.

/close
